src/sql/workbench/contrib/executionPlan/browser/executionPlanViewHeader.ts

```diff
@@ -88,6 +88,7 @@ export class ExecutionPlanViewHeader {
 	private renderQueryText(): void {
 		if (this._query) {
 			this._queryContainer.innerText = this._query;
+			this._queryContainer.title = this._query;
```

Comment:
This makes it show the full query as a tooltip right?

Is there a way for the user to copy the query text easily? Might be a feature to consider adding later

Reply:
They have an option to open query in a new query window for now. 😃